ui/app/i18n/clinical/locale_fr.json

```diff
@@ -277,5 +277,6 @@
   "PRESCRIPTION_REPORT_PRINT_PRESCRIPTION_LABEL": "Imprimer la préscription",
   "PRESCRIPTION_REPORT_PRINT_TARV_PRESCRIPTION_LABEL": "Imprimer la prescription TARV",
   "CLINICAL_ADD_AS_CONDITION":"Ajouter comme condition",
-  "PRESCRIPTION_REPORT_LABEL_FOR": "pour"
+  "PRESCRIPTION_REPORT_LABEL_FOR": "pour",
+  "PROGRAM_MANAGEMENT_LABEL_THERAPEUTIC_LINE":"Ligne thérapeutique"
```

Comment:
@IipperMangesh PROGRAM_MANAGEMENT_CHOOSE_THERAPEUTIC_LINE french translation is missing here.

Reply:
@lauravignoli  PROGRAM_MANAGEMENT_CHOOSE_THERAPEUTIC_LINE  is not needed, I just delete this key from the English translation.